demos/helpers.sh

```diff
@@ -39,3 +39,26 @@ comment() {
   echo -e "\033[0;32m${indented}\033[0m"
   blank
 }
+
+git_config() {
```

Comment:
Why not also export `JJ_CONFIG` and `GIT_CONFIG_GLOBAL` here? Do you see a use for running the scripts as the current user?

Reply:
I am not sure whether that export will persist outside the function/helper script. I'm not sure what the rules are, I only know that they are weird (round brackets and curly brackets differ).

Reply:
Yes, when you source it (as we do in each test), it's run in the caller's context, so it will work

Reply:
I created a separate file to be loaded with `. setup_standard_config.sh`. I feel like it's simple enough and that I might want to occasionally try the demos out with some new config.